src/Bowling.js

```diff
@@ -0,0 +1,101 @@
+function Bowling() {
+  this.frameCount = 1;
```

Comment:
something like `frameCount` should be dynamically counted rather than tracked as state